services/dask-sidecar/requirements/_test.in

```diff
@@ -15,7 +15,6 @@ pytest-aiohttp  # incompatible with pytest-asyncio. See https://github.com/pytes
 pytest-cov
 pytest-icdiff
 pytest-instafail
-pytest-lazy-fixture
```

Comment:
:tada: should we add it to the block list?

Reply:
not sure it is really needed.